qa/tasks/qemu.py

```diff
@@ -87,8 +88,24 @@ def generate_iso(ctx, config):
     """Execute system commands to generate iso"""
     log.info('generating iso...')
     testdir = teuthology.get_testdir(ctx)
+
+    refspec = ctx.config.get('branch')
```

Comment:
Actually, that test ran on a previous version which was using just `config`. The change to `ctx.config` is trying to fix that. I think the sha1 item was somehow getting squashed by https://github.com/ceph/ceph/blob/master/qa/tasks/qemu.py#L444 - i.e. immediately prior to the `generate_iso()` lambda definition.